.gitignore

```diff
@@ -0,0 +1,2 @@
+.*.swp
```

Comment:
Hell, I don't even know what they are. Looks like that line came in a recent PR that I (apparently) didn't review closely enough.